kin-devplatform-sdk/build.gradle

```diff
@@ -19,8 +19,9 @@ android {
 dependencies {
 
     implementation project(':kin-devplatform-core')
+    implementation 'com.github.kinecosystem:kin-migration-module-android:e4143f7358'
     implementation 'com.github.kinecosystem.kin-ecosystem-android-sdk:recovery:d73ef56'
-    implementation devDependencies.kinCore
+    api 'com.github.kinecosystem:kin-utils-android:1.1'
```

Comment:
move to `dependencies.gradle` ?
And why you deleted kinCore ?

Reply:
dependencies come from the migration module, that has both kin-core and kin-sdk as a dependency, and abstract away the details when core/sdk will be used.

Reply:
I don't understand, you still using `dependencies.gradle` why not have all the deps there?

Reply:
Because we don't like the dependencies.gradle file and we are starting step by step to put everything back to the gradle file.
Anyway before it was also not in the dependencies.gradle file
Alos this wasn't: implementation 'com.github.kinecosystem.kin-ecosystem-android-sdk:recovery:d73ef56'